content/applications/sales/point_of_sale/shop/salesorder.rst

```diff
@@ -0,0 +1,45 @@
+==================
+Sales order in PoS
+==================
+
+Point of sale management can expose businesses to certain difficulties, such as the lack of a
+particular product in-store. To prevent losing a potential client, they need a way to order products
+directly from their shop.
+
+Fortunately, the point of sale application is fully integrated with the sales application, meaning
+that one can create a sales order and find it in the pos. Then, they can pay for it, either
+partially or completely, from the pos.
+
+Select a sales order
+====================
+
+From the **Point of Sale** application, open a new session. Then, click on
+:guilabel:`Quotations/Orders` to get the complete list of quotations and sales orders created on the
+sales application.
+
+.. image:: salesorder/salesorder_pos-interface.png
+   :align: center
+   :alt: Quotations and order button on the Point of Sale interface
+
+.. note::
+   To ease finding the right sales order, you can filter that list on the **customer** or on the
+   **order reference**. You can also set the customer before clicking on
+   :guilabel:`Quotations/Orders` to reduce the list to one particular customer.
+
+Apply a down payment or settle the order
+========================================
+
+Click on the sales order to make a payment. You can either:
+
+- Settle the order **partially**: after you click on :guilabel:`Apply a down payment`, enter the
+  percentage of the amount that the customer is paying. Then, click on "ok" and proceed with the
+  order.
+- Settle the order **completely**: click on :guilabel:`Settle the order` to pay for the total of the
+  sales order.
+
+.. note::
+   Once you settle a sales order, the applied down payment is automatically deduced from the total
+   amount.
+
+.. Seealso::
+   - :ref:`Down payments documentation <down_payment/down_payment>`
```

Comment:
It's better to use the `doc` role to link to entire doc pages so that:
- The page title can be used as the label.
- The page is not scrolled to the custom anchor.